doc/cla/individual/TeoGoddet.md

```diff
@@ -0,0 +1,9 @@
+Switzerland, 2021-12-11
```

Comment:
This should be in another commit, with the [CLA] tag https://www.odoo.com/documentation/15.0/developer/misc/other/guidelines.html#tag-and-module-name